src/components/Contentful/DynamicHeroClassic.vue

```diff
@@ -0,0 +1,335 @@
+<template>
+	<section-with-background-classic
+		:background-content="background"
+		:vertical-padding="verticalPadding"
+	>
+		<template #content>
+			<kv-page-container>
+				<kv-grid
+					class="tw-grid-cols-12 tw-gap-2 tw-items-center"
+					:class="{ 'tw-grid-cols-1 tw-mx-auto': singleColumn }"
+					:style="customGridStyles"
```

Comment:
This is a garbage pail of styles that are easier to manage directly rather than using tailwind classes. We start with options for max width and text alignment to support the "It's a Loan not a donation" layout.

Reply:
These are NOT automatic, the will require configuration to setup and apply with any new styles we want to pass from Contentful.